### PR TITLE
PHPLIB-326: Aggregation, runCommand, and index management examples for Docs

### PR DESCRIPTION
**Pull request by kvwalker**
https://jira.mongodb.org/browse/PHPLIB-326

### CODE REVIEW

File: tests/DocumentationExamplesTest.php [hunk 1]
```diff
@@ -1022,6 +1029,128 @@ public function testChangeStreamExample_1_4()
...
+                ]
+            ]]
+        ]);
+        $this->assertInstanceOf('MongoDB\Driver\Cursor', $cursor);
+    }
+
+    public function testAggregation_example_4()
+    {
+        if (version_compare($this->getServerVersion(), '3.6.0', '<')) {
+            $this->markTestSkipped('let of an object is not supported');
```
Comment (jmikola) [inline tests/DocumentationExamplesTest.php]: Is there a citation for this? I didn't see anything mentioned in the [3.6 release notes](https://docs.mongodb.com/manual/release-notes/3.6/) and [`$let` documentation](https://docs.mongodb.com/manual/reference/operator/aggregation/let/) doesn't talk about version differences.

File: tests/DocumentationExamplesTest.php [hunk 2]
```diff
@@ -1022,6 +1029,128 @@ public function testChangeStreamExample_1_4()
...
+                ]
+            ]]
+        ]);
+        $this->assertInstanceOf('MongoDB\Driver\Cursor', $cursor);
+    }
+
+    public function testRunCommand_example_1()
+    {
+        $db = new Database($this->manager, $this->getDatabaseName());
+        $cursor = $db->command(['buildInfo' => 1]);
```
Comment (jmikola) [inline tests/DocumentationExamplesTest.php]: For illustrative purposes, I might suggest adding `$result = $cursor->toArray()[0];` on the subsequent line. This is what we currently do in https://docs.mongodb.com/php-library/current/tutorial/commands/.

File: tests/DocumentationExamplesTest.php [hunk 3]
```diff
@@ -1022,6 +1029,128 @@ public function testChangeStreamExample_1_4()
...
+        $this->assertInstanceOf('MongoDB\Driver\Cursor', $cursor);
+    }
+
+    public function testRunCommand_example_2()
+    {
+        $db = new Database($this->manager, $this->getDatabaseName());
+        $cursor = $db->command(['collStats' => 'restaurants']);
```
Comment (jmikola) [inline tests/DocumentationExamplesTest.php]: Likewise suggestion: `$result = $cursor->toArray()[0];` below this.

File: tests/DocumentationExamplesTest.php (hunk 1 above)
Comment (kvwalker) [reply tests/DocumentationExamplesTest.php]: This test fails on 3.2 and 3.4 with `arguments to $lookup must be strings, let: { constituents: "$airlines" } is type object`. [This](https://docs.mongodb.com/manual/reference/operator/aggregation/lookup/index.html#join-conditions-and-uncorrelated-sub-queries) is the only documentation I could find that might explain why.

**Comment by jmikola** [comment]
Changed to `v1.3` for historical accuracy. Thanks for merging this to the correct branch! I didn't realize it was `master` during the review period.

**Pull request closed without merging**